### DISCUSSION TITLE
Is it possible to get a list of followers of my repo via an API call?

### DISCUSSION THREAD

**Discussion by DataJuggler** [community/community#24430] [Apps, API and Webhooks]
<p>I just released an open source project for SQL Server developers called <a href="http://DataTier.Net" rel="nofollow noopener">DataTier.Net</a> (formerly it was called RAD Studio Code Generation Toolkit on CodePlex). I bought the domain <a href="http://DataTier.Net" rel="nofollow noopener">DataTier.Net</a> last year, so I thought an interesting and very simple sample project would be to create a website with a 1 table SQL Dabase as a sample project and let people sign up for email notifications when a new version is released on Github. I never even checked if GitHub has this functionality built in? If it does it doesn’t matter this is a sample project.</p>
<p>Since this is a simple site with unauthenticated users, rather then the let the public at large sign up for email notifications, I thought if the user entered both an email address, and a GitHub user name that followed me on GitHub this would slow down any bots or bored peopled from just typing in email addresses and make me out to be a spammer. As of now I could hard code all 2 followers, but I am hoping one day to have as many as 6 or 7 and hard coding isn’t the answer, or I might lose one and have to update the site again for maintenance.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/cba8bb09ea32e7529a85ad233685eb1a43a3358e.jpeg" data-download-href="/uploads/short-url/t3EqFDGRYyWziK29hX5c7Z49TUa.jpg?dl=1" title="API Followerse.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/26331133/181084715-485a2d7d-2631-4aed-8e06-37dfd7de6a9f.jpg" alt="API Followerse.png" data-base62-sha1="t3EqFDGRYyWziK29hX5c7Z49TUa" width="657" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/cba8bb09ea32e7529a85ad233685eb1a43a3358e_2_657x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/c/cba8bb09ea32e7529a85ad233685eb1a43a3358e_2_985x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/c/cba8bb09ea32e7529a85ad233685eb1a43a3358e_2_1314x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/cba8bb09ea32e7529a85ad233685eb1a43a3358e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">API Followerse.png</span><span class="informations">1384×1053 605 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks if this possible,</p>
<p>Corby</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You can list the users that have “starred” your repo by using the <a href="https://developer.github.com/v3/activity/starring/#list-stargazers" rel="nofollow noopener">List Stargazers API endpoint</a>.</p>
<p>I hope that helps and let us know if you have any additional questions.</p>

**Comment by DataJuggler** [comment]
<p>Thanks. That means I have to change my sample project and video I already made, but I made a few mistakes I caught when I rewatched it due to filming at 5 AM.</p>
<p>This weekend I will see if I can figure it out.</p>
